apps/src/gamelab/GameLab.js

```diff
@@ -404,7 +403,8 @@ GameLab.prototype.init = function (config) {
     config.initialAnimationList : this.startAnimations;
   getStore().dispatch(setInitialAnimationList(initialAnimationList));
 
-  const loader = this.loadLibraries_().then(() => ReactDOM.render((
+  this.loadValidationCodeIfNeeded_();
+  const loader = this.studioApp_.loadLibraries_(this.level.helperLibraries).then(() => ReactDOM.render((
```

Comment:
Nit: Maybe `loadLibraries_` shouldn't use a private method naming convention, if we expect it to be called from other contexts.